manifests/init.pp

```diff
@@ -6,6 +6,7 @@
   $couchdb_data_directory = $nunaliit::params::couchdb_data_directory,
   $atlas_parent_directory = $nunaliit::params::atlas_parent_directory,
   $atlas_source_directory = $nunaliit::params::atlas_source_directory,
+  $basic_auth = $nunaliit::params::basic_auth
```

Comment:
What is the basic_auth parameter in nunaliit? So if I wanted to set the basic_auth parameter, I would set it in the manifests/params.pp file?
I'm sorry I don't have context here. How is basic authentication different from what happens in the usual nginx configuration?

Reply:
Basic auth is the pop-up you get asking for a username and password. ELOKA wanted to setup an atlas that is password protected so only specific people can access that atlas. Generally Nunaliit only support login for modification/admin controls. In this case they didn't want anyone without permission to be able to see the content, so this puts a wall in front of the whole app.

This is written to be backwards compatible so if the flag is not set there will be not basic auth. You set it in nunaliit-common.yaml. See https://bitbucket.org/nsidc/nunaliit/src/519f17ca82fca11683a2e2beb1c626fddc257b37/puppet/nunaliit-common.yaml?at=snowy&fileviewer=file-view-default
for an example.

Reply:
OK, thanks. This helps me to understand the whole "nunaliit-common.yaml" thing.